en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md

```diff
@@ -0,0 +1,168 @@
+     
+    6. Now you have successfully created an Azure app. Double click the app and you will see its details as below.**Copy** the Value of **Application (client) ID** and **Directory (tenant) ID**.
+        
+       <img src="/assets/img/connectors/application-id.png" title="Application ID" width="800" alt="Application ID"/>
+  
+    7. We need to give permission to the app to access Dynamics 365. Navigate to **View API permissions**. 
+    
+       <img src="/assets/img/connectors/view-api-permissions.png" title="View API permissions" width="800" alt="View API permissions"/>
+    
+    8. Click **Add a permission**. 
+    
+       <img src="/assets/img/connectors/add-permission.png" title="Add a permission" width="800" alt="Add a permission"/>  
+           
+    9. Then select **Dynamics CRM**.     
+    
+       <img src="/assets/img/connectors/select-api.png" title="Select an API" width="800" alt="Select an API"/>
+       
+    10. Make sure to check the Delegated Permissions checkboxes as shown below.**Select permissions** and click **Add permission**.
+     
+        <img src="/assets/img/connectors/select-permission.png" title="Select CRM permission" width="800" alt="Select CRM permission"/>
+       
+    11. Click on **Grant Permissions** and click **Yes**.   
+       
+        <img src="/assets/img/connectors/grant-required-permission.png" title="Grant permission" width="800" alt="Grant permission"/>
+       
+    12. After setup CRM permissions you will see following console.   
+    
+        <img src="/assets/img/connectors/after-setup-permission.png" title="After setup CRM permissions" width="800" alt="After setup CRM permissions"/> 
+    
+    13. Then need create secret keys. Navigate to **Certificates & secrets**
+        
+        <img src="/assets/img/connectors/certificate-and-secrets.png" title="Create certificate and secrets" width="800" alt="Create certificate and secrets"/> 
+    
+    14. Click new **New client secrets**
+    
+        <img src="/assets/img/connectors/new-client-secret.png" title="New client secret" width="800" alt="New client secret"/> 
+     
+    15. Add **Description** and **Expires** values. Click **Add** and copy the value.    
+    
+        <img src="/assets/img/connectors/add-client-secret.png" title="Add client secret" width="800" alt="Add client secret"/>
+    
+    16. Search **Users** inside the Azure Active Directory and Create **New user**.(This user would be linked to the Application User which will create in the Dynamics 365 CRM)    
+        
+        <img src="/assets/img/connectors/add-new-user.png" title="Create new user" width="800" alt="Create new user"/>
+        
+    17. Make sure fill all mandatory fields and click **Create**.    
+       
+        <img src="/assets/img/connectors/create-user.png" title="Fill new user details" width="800" alt="Fill new user details"/>
+
+* ## Setting Up the Application user in Microsoft Dynamics 365 CRM
```

Comment:
```suggestion
## Setting Up the Application user in Microsoft Dynamics 365 CRM
```

Reply:
"Setting Up an App in Azure" and "Setting Up the Application user in Microsoft Dynamics 365 CRM" topics should be under "Authentication to Dynamics 365 using Azure Apps topic". I think it is better to maintain above topics as two bullet points under Authentication to Dynamics 365 using Azure Apps.